x-pack/plugins/security_solution/public/management/pages/trusted_apps/view/trusted_apps_page.test.tsx

```diff
@@ -555,7 +563,7 @@ describe('When on the Trusted Apps Page', () => {
           // to test the UI behaviours while the API call is in flight
           coreStart.http.post.mockImplementation(
             // @ts-ignore
-            async (path: string, options: HttpFetchOptions) => {
+            async (_path: string, options: HttpFetchOptions) => {
```

Comment:
Out of curiosity, why did you change this `path` to `_path` ?

Reply:
to silence the ts-warning about non used parameter.

Reply:
That make sense. Personally I prefer to set an underscore without type since it's not going to be used, but both works in the same way:

```
async (_, options: HttpFetchOptions) => {
```

Thanks for fixing this warning 🤗

Reply:
I decided to change it in this PR